includes/bp-members/classes/class-bp-members-endpoints.php

```diff
@@ -0,0 +1,402 @@
+<?php
```

Comment:
Did u see my comment in slack re: keeping with the standards of the BP-REST master in case I eventually PR ?